stable/firehose/templates/deployment.yaml

```diff
@@ -85,3 +85,11 @@ spec:
       - emptyDir: {}
         name: workdir
       {{- end }}
+      {{- if index .Values "taints" "enabled" }}
```

Comment:
why not `{{ - if index .Values.taints.enabled }}` ?

Reply:
sure, can do this